lua/conform/formatters/pint.lua

```diff
@@ -0,0 +1,15 @@
+local util = require("conform.util")
+
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/laravel/pint",
+    description = "Laravel Pint is an opinionated PHP code style fixer for minimalists. Pint is built on top of PHP-CS-Fixer and makes it simple to ensure that your code style stays clean and consistent.",
```

Comment:
The README is generated from this description, so I think we can keep just the first sentence

Reply:
Done!